backend/server/tests/calculateAchievementStatus.test.ts

```diff
@@ -0,0 +1,26 @@
+import('../core/interfaces/challenge.interface');
+import { StateItem } from '../core/enums/stateItem.enum';
+import { Status } from '../core/interfaces/status.interface';
+import { Achievement } from '../core/interfaces/achievement.interface';
+import { loadAchievements } from '../../data/achievements';
+import { calculateAchievementsStatus } from '../core/services/calculateAchievementsStatus.service';
+
+describe('Test calculateAchievementsStatus', () => {
+  it('should test that calculateAchievementsStatus calculates expected achievementsStatus', () => {
```

Comment:
Don't use "should test" in descriptions. Just `it('should calculates expected achievementsStatus', ...`